fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -1053,7 +1053,9 @@ public static DataCacheInfo analyzeDataCacheInfo(Map<String, String> properties)
         if (!enableDataCache && enableAsyncWriteBack) {
             throw new AnalysisException("enable_async_write_back can't be turned on when cache is disabled");
         }
-
+        if (enableAsyncWriteBack) {
```

Comment:
can we just prompt the user a warning message rather than error?

the same SQL from 3.0 can't be run in a v3.1 cluster, and the same 3.1 SQL can't run on a v3.2 cluster, feel frustrating if I am the user.

Reply:
good ideal

Reply:
Very unfortunately the current framework does not support to prompt a warning on create table ddl

Reply:
why disable `enable_async_write_back` property?

Reply:
enable_async_write_back may loss data, which is not what we want to see

Reply:
ok, thanks